kvdb/src/main.rs

```diff
@@ -168,7 +165,7 @@ pub(crate) enum KvdbDumpMode {
 }
 
 #[derive(Clone, Debug, Parser)]
-#[clap(rename_all = "kebab-case")]
+#[clap(author, version, about = "Examine Polkadot and Substrate key-value databases", rename_all = "kebab-case")]
```

Comment:
I think this tool would work regardless of the software that crates the DBs.
```suggestion
#[clap(author, version, about = "Examine RocksDB and ParityDB databases", rename_all = "kebab-case")]
```